crates/sui-graphql-rpc/src/types/move_registry/named_type.rs

```diff
@@ -145,40 +152,47 @@ mod tests {
         let mut demo_data = vec![];
 
         demo_data.push(DemoData {
-            input_type: "app@org::type::Type".to_string(),
+            input_type: "@org/app::type::Type".to_string(),
+            expected_output: format_type("0x0", "::type::Type"),
+            expected_names: vec!["@org/app".to_string()],
+        });
+
+        demo_data.push(DemoData {
+            input_type: "inner@org/app::type::Type".to_string(),
             expected_output: format_type("0x0", "::type::Type"),
-            expected_names: vec!["app@org".to_string()],
+            expected_names: vec!["inner@org/app".to_string()],
         });
 
         demo_data.push(DemoData {
-            input_type: "0xapp@org::type::Type".to_string(),
+            input_type: "@org/0xapp::type::Type".to_string(),
             expected_output: format_type("0x0", "::type::Type"),
-            expected_names: vec!["0xapp@org".to_string()],
+            expected_names: vec!["@org/0xapp".to_string()],
         });
 
         demo_data.push(DemoData {
-            input_type: "app@org::type::Type<u64>".to_string(),
+            input_type: "@org/app::type::Type<u64>".to_string(),
             expected_output: format!("{}<u64>", format_type("0x0", "::type::Type")),
```

Comment:
nit: What happens if you incorporate `<u64>` into the last parameter of `format_type`?

```suggestion
            expected_output: format_type("0x0", "::type::Type<u64>"),
```

Reply:
Nothing, not sure why I did that 😆